crates/bitwarden-generators/src/passphrase.rs

```diff
@@ -1,11 +1,15 @@
 use bitwarden_crypto::EFF_LONG_WORD_LIST;
+use bitwarden_error::bitwarden_error;
 use rand::{seq::SliceRandom, Rng, RngCore};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 use thiserror::Error;
+#[cfg(feature = "wasm")]
+use {tsify_next::Tsify, wasm_bindgen::prelude::*};
```

Comment:
I don't think the `wasm_bindgen` import is needed here, as we're not using it directly. I'm kind of surprised it's not giving out a warning though 🤔 
```suggestion
use tsify_next::Tsify;
```